assets/new-user-experience/modules/advanced-settings/index.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies.
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { ToggleControl } from '@wordpress/components';
+import parse from 'html-react-parser';
+
+/**
+ * Internal dependencies.
+ */
+import {
+	Section,
+	SectionTitle,
+	SectionDescription,
+	InputWrapper,
+	SquareCheckboxControl,
+} from '../../components';
+import { DebugMode } from '../../modules';
+import { useSquareSettings } from '../../settings/hooks';
+import { useSteps } from '../../onboarding/hooks';
+
+export const AdvancedSettings = () => {
+	const {
+		settings,
+		squareSettingsLoaded,
+		setSquareSettingData,
+	} = useSquareSettings();
+
+	const {
+		stepData: {
+			step,
+		}
+	} = useSteps();
+
+	const {
+		enable_customer_decline_messages,
+		debug_logging_enabled = 'no',
+	} = settings;
+
+	if ( ! ( squareSettingsLoaded ) ) {
```

Comment:
Do we need the extra parentheses here?
```suggestion
	if ( ! squareSettingsLoaded ) {
```